### DISCUSSION TITLE
multi_ep and multi_recv not working over verbs

### DISCUSSION THREAD

**Discussion by JibinNajeeb** [ofiwg/libfabric#8299] [Q&A]
I am checking for a way to retrieve buffers from multiple senders to a single receiver. 

I have checked the sample fabtest apps(multi_ep and multi_recv) and getting error as shown below

![multi_ep](https://user-images.githubusercontent.com/47626138/205136345-05491ac1-4b26-410d-b243-8120296a83f1.PNG)
![multi_recv](https://user-images.githubusercontent.com/47626138/205136393-bf69f122-0f6a-4cc1-9253-b1fd8917f68c.PNG)

Is there a way to achieve this behavior -> get buffers from multiple client(4  clients) and retrieve it in server?

**Comment by shefty** [comment]
It sounds like you may want to use rdm endpoints.  That provides unconnected communication, so a single server endpoint can receive from multiple clients.  Using msg endpoints requires managing multiple connections, one to each client.

**Comment by JibinNajeeb** [comment]
Receiving 256 error when trying with rdm
![image](https://user-images.githubusercontent.com/47626138/205218998-2ee7ef3f-b226-4160-a263-8444ab84afd6.png)
